modules/moderation.py

```diff
@@ -268,350 +331,203 @@ async def _unmuting(self, ctx, member: discord.Member, *, reason='-No reason spe
-        db = mclient.bowser.puns
-        warnPun = db.find_one({'user': member.id, 'active': True, 'type': {
-                    '$in': [
-                        'tier1',
-                        'tier2',
-                        'tier3'
-                    ]
-                }
-            }
-        )
-
-        if not warnPun:
-            return await ctx.send(f'{config.redTick} No warnings are currently active for {str(member)} ({member.id})')
-
-        issueTime = datetime.datetime.utcfromtimestamp(warnPun['timestamp']).strftime('%B %d, %Y %H:%M:%S UTC')
-
-        embed = discord.Embed(title="Warning review", colour=discord.Color(0xea4345), description="To change the status of the warning, react with the following emoji. You must react to make a choice.\n\n:track_next: Re-review in 30 days\n:fast_forward: Re-review in 14 days\n:arrow_forward: Re-review in 7 days\n:small_red_triangle_down: Reduce warn tier (or remove if tier1)\n:octagonal_sign: Make warning permanent", timestamp=utils.resolve_duration('15m'))
-        embed.set_thumbnail(url=member.avatar_url)
-        embed.set_author(name=f"{member} ({member.id})")
-        embed.set_footer(text="This message will expire in 15 minutes")
-        embed.add_field(name="Warning details", value=f'**Type:** {config.punStrs[warnPun["type"]]}\n**Issued by:** <@{warnPun["moderator"]}>\n**Issued at:** {issueTime}\n**Reason**: {warnPun["reason"]}')
-
-        resp = await ctx.send(embed=embed)
-        await resp.add_reaction(config.nextTrack) # track_next
-        await resp.add_reaction(config.fastForward) # fast_forward
-        await resp.add_reaction(config.playButton) # arrow_forward
-        await resp.add_reaction(config.downTriangle) # small_red_triangle_down
-        await resp.add_reaction(config.stopSign) # octagonal_sign
-
-        metaReactions = {
-            config.nextTrack: 0,
-            config.fastForward: 0,
-            config.playButton: 0,
-            config.downTriangle: 0,
-            config.stopSign: 0
-        }
-        tierLevel = {
-            'tier1': ctx.guild.get_role(config.warnTier1),
-            'tier2': ctx.guild.get_role(config.warnTier2),
-            'tier3': ctx.guild.get_role(config.warnTier3)
-        }
-        renew = False
-        perm = False
+        if await tools.mod_cmd_invoke_delete(ctx.channel):
+            return await ctx.message.delete()
 
-        def check(reaction, user):
-            if user.bot:
-                return False
-            #print(reaction.emoji)
+        await ctx.send(f'{config.greenTick} {member} ({member.id}) has been successfully struck')
 
-            if ctx.guild.get_role(config.moderator) in user.roles and str(reaction.emoji) in [config.nextTrack, config.fastForward, config.playButton, config.downTriangle, config.stopSign]:
-                metaReactions[str(reaction.emoji)] += 1
-                #print(metaReactions)
-                if metaReactions[str(reaction.emoji)] >= 1:
```

Comment:
Probably should remove old prints to clean

Reply:
This line was removed in this PR 🤦